src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,38 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    protected static final int PASSWORD_MIN_LENGTH = 6;
+    protected static final int AGE_BOUND = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User is null.");
+        }
+        if (user.getLogin() == null) {
+            throw new RuntimeException("User login is null.");
+        }
+        if (user.getPassword() == null) {
+            throw new RuntimeException("Password login is null.");
+        }
+        if (user.getAge() == null) {
+            throw new RuntimeException("Age login is null.");
+        }
+        if (user.getAge() < 0) {
+            throw new RuntimeException("Age must be positive");
+        }
```

Comment:
This check is unnecessary, because if the age is negative it will be less than 18

Reply:
Done.